src/lib/components/file-browser/file-list/file-list.component.html

```diff
@@ -1,104 +1,191 @@
-<terra-base-toolbar>
-    <div class="btn-group" role="group">
-        <terra-button
-            inputIcon="icon-upload"
-            [inputTooltipText]="_translationPrefix + '.uploadFile' | translate: _locale.language"
-            inputTooltipPlacement="right"
-            (outputClicked)="fileChooser.click()"
-        >
-        </terra-button>
-        <terra-button
-            inputIcon="icon-folder_add"
-            *ngIf="_parentFileBrowser.inputAllowFolders"
-            [inputTooltipText]="_translationPrefix + '.createFolder' | translate: _locale.language"
-            inputTooltipPlacement="right"
-            (outputClicked)="_showNewDirectoryPrompt = true"
-        >
-        </terra-button>
-    </div>
-
-    <div class="btn-group" role="group">
-        <terra-button
-            inputIcon="icon-delete"
-            [inputIsDisabled]="_selectedStorageObjects.length <= 0"
-            [inputTooltipText]="_translationPrefix + '.deleteSelectedFiles' | translate: _locale.language"
-            inputTooltipPlacement="right"
-            (outputClicked)="_deleteSelected()"
-        >
-        </terra-button>
-    </div>
+<mat-toolbar>
```

Comment:
hier fehlt noch `<div class="terra-table-container">` drum herum (in Terra gibts da ne Directive dafür, die das macht), ausserdem kannst du `terra-table-toolbar` als class auf `mat-toolbar` machen

dann sollten auch die styles stimmen